docs/02-membership.md

```diff
@@ -276,6 +249,78 @@ The table and graphs below illustrate how the 5 year weight change of a hypothet
+
+4. Incentive compatibility
+
+    - It is incentive compatible that curators (Guild members) are drawn from the beneficiaries (Guild members).
+        - Adding beneficiaries removes future vested value from existing members. They will more carefully consider potential members and their contributions. An external council would not feel this constraint so directly.
+    - The mechanism must accept all legitimate contributors
+        - This prevents the set from ossifying or getting captured. Potential members which fit established guidelines need to be added to maintain credible neutrality to participants and sponsors. If donors think that the set is not curated well enough, they will not feel incentivized to contribute.
+
+### Adding a Member
+
+An existing member should make a PR at [this repo](https://github.com/protocolguild/documentation) which proposes an addition to the member list above, along with the accompanying info:
+
+- Name / Identifier
+- Affiliated project they work on which is eligible
+- Links to relevant eligible work, eg. GitHub, research
+- 3-4 sentence summary of their contributions
+
+See some examples here: [open](https://github.com/protocolguild/documentation/pulls) or [closed](https://github.com/protocolguild/documentation/pulls?q=is%3Apr+is%3Aclosed+label%3A%22add+member%22) PRs
+
+Discussion should be open for at least one week to give members time to review and discuss.
+
+Bias or conflicts of interest of the nominator should be disclosed, if they exist, e.g. where one is an advisor to the other’s side project. After one week of rough consensus, the PR should be merged or rejected.
+
+### Removing a Member
+
+#### Self-Removal, Affiliation, Weights
+
+Removals from the active membership should come from the member themselves, i.e. self-removal, by opening a PR in the [same repo](https://github.com/protocolguild/documentation). Example [here](https://github.com/protocolguild/documentation/pull/152). Where this isn't possible due to extenuating circumstances, the member should be notified or tagged on the PR so they are aware of the changes.
+
+Affiliation and weight changes should include some rationale for the change, ideally from the member themselves and seconding by a colleague.
+
+#### Peer Removal
+
+Peer removals can occur when a member stops contributing to eligible work, and is not responsive to requests to self-remove (as described above). In such cases, another member (ideally from the same team) can propose to remove the inactive member.
+
+There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample references to the alleged misconduct and justification for the removal. To date, this method has never been used.
```

Comment:
```suggestion
There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce. The PR should include ample documentation and justification for the removal. To date, this method has never been used.
```

I think removing the "misconduct" here allows this to be more generally applicable since I believe it's reasonable to expect there to be reasons beyond "misconduct" that might arise.

Reply:
@macfarla I think you can make a case that "higher" is also correct, and @pipermerriam I agree with your point re: broadening. My takeaway after re-reading the paragraph 3 times is we should just remove that sentence to simplify things. 

```suggestion
There may also be situations when another member proposes the removal of an existing member, even if they are continuing with eligible work. This should only happen in special circumstances where the removed member's contributions are far below what would generally be expected, or in the case of other grave misconducts. The PR should include ample documentation and justification for the removal. To date, this method has never been used.
```

Reply:
yeah, the wording could be improved